docs/translations/api-docs/menu/menu.json

```diff
@@ -10,6 +10,8 @@
     "onClose": "Callback fired when the component requests to be closed.<br><br><strong>Signature:</strong><br><code>function(event: object, reason: string) =&gt; void</code><br><em>event:</em> The event source of the callback.<br><em>reason:</em> Can be: <code>&quot;escapeKeyDown&quot;</code>, <code>&quot;backdropClick&quot;</code>, <code>&quot;tabKeyDown&quot;</code>.",
     "open": "If <code>true</code>, the component is shown.",
     "PopoverClasses": "<code>classes</code> prop applied to the <a href=\"/material-ui/api/popover/\"><code>Popover</code></a> element.",
+    "slotProps": "The extra props for the slot components. You can override the existing props or add new ones.",
```

Comment:
We have a merge conflict regarding this file, this is because we recently updated our API docs format: https://github.com/mui/material-ui/pull/37405

Here are the steps I would follow to solve this:

1. Go to the `master` branch and `git pull upstream master`
2. Checkout the `MUI-37612` branch
3. Revert this `api-docs/menu/menu.json` change locally, this is so step 4. doesn't fail due to the merge conflict. There's no issue in reverting as we will generate again the changes in step 5.
4. Merge master into `MUI-37612`: `git merge --no-ff master`
5. Run `yarn docs:api` again. This should bring back the `slotProps` and `slots` entries with the new format.